internals/plan/plan.go

```diff
@@ -796,35 +764,32 @@ func CombineLayers(layers ...*Layer) (*Layer, error) {
 			}
 		}
 
-		switch target.Selection {
-		case OptOutSelection, OptInSelection, DisabledSelection, UnsetSelection:
-			// valid, continue
-		default:
+		// Validate service names specified in log target
+		for _, serviceName := range target.Services {
+			if serviceName == "all" || serviceName == "-all" {
+				continue
+			}
+			if _, ok := combined.Services[serviceName]; ok {
+				continue
+			}
+			// Check if this is `-svc`
+			serviceName = strings.TrimPrefix(serviceName, "-")
+			if _, ok := combined.Services[serviceName]; ok {
```

Comment:
It seems to me that only this second `combined.Services` check is needed. First trim any `-` prefix, then check if it's a service name. No need for the first one above.

Reply:
good catch

Reply:
We also need to disallow service names starting with `-`